robot/basestation/static/js/elements/feed/camera.js

```diff
@@ -135,7 +138,6 @@ $(document).ready(() => {
     cameraPower.attr('src', POWER_ON)
 
     cameraControls.attr('camera-name', cameraName)
-    console.log("memes")
```

Comment:
lmfao I'm dead @RealWilliamWells @PeterGhimself @vashmata @AFknUsername

Reply:
lol, ngl that was prob me. I always print "memes" if I'm using console.log to test